src/controller/adc.h

```diff
@@ -17,10 +17,21 @@
 #define UI8_ADC_THROTTLE 				      (*(uint8_t*)(0x53EE)) // AIN7
 #define UI8_ADC_TORQUE_SENSOR         (*(uint8_t*)(0x53E8)) // AIN4
 
+#define UI16_ADC_10_BIT_BATTERY_VOLTAGE 	(((*(uint8_t*)(0x53EC)) << 2) | (*(uint8_t*)(0x53ED)))
```

Comment:
I'd suggest adding a cast (uint16_t)(...) over the final value, such that if you try to assign it to an uint8_t, the compiler will warn you.

Reply:
Great suggestion! However not sure if this define will be used. Will keep it in mind nonetheless, thanks!